cloud/kubernetes/helm/yugabyte/values.yaml

```diff
@@ -49,6 +50,7 @@ Services:
       rpc-port: "7100"
       yql-port: "9042"
       yedis-port: "6379"
+    memory_limit_to_ram_ratio: 1
```

Comment:
The default (for TServers) is already 0.85 and we don't want to change that. We want to leave some room for other processes.

Reply:
In other words, let us not override the TServer setting for memory limit in this yaml file.